File: src/dart/prime_number.dart
bool isPrime(int num) {
  if (num <= 1)
    return false;
  else if (num == 2 || num == 3)
    return true;
  else if (num % 2 == 0 || num % 3 == 0)
    return false;
  else
    for (var i = 5; i * i < num; i += 6)
      if (num % i == 0 || num % (i + 2) == 0) return false;
  return true;
}

File: src/C++/PrimeNumber.cpp
#include <bits/stdc++.h>
using namespace std;

bool isPrime(int n)
{

	for (int divisor = 2; divisor * divisor <= n; divisor++)
	{
		if (n % divisor == 0)
		{
			return false;
		}
	}

	return (n >= 2);
}
